encord/user_client.py

```diff
@@ -602,6 +604,40 @@ def __validate_filter(self, properties_filter: Dict) -> Dict:
 
         return ret
 
+    def deidentify_dicom_files(
+        self,
+        dicom_urls: List[str],
+        integration_hash: str,
+    ) -> List[str]:
+        """
+        Deidentify dicom series in external storage.
+
+        Args:
+            self: Encord client object.
+            dicom_urls: a list of urls to DICOM files, e.g.
+                ['https://s3.region-code.amazonaws.com/bucket-name/dicom-file.dcm']
+            integration_hash:
+                integration_hash parameter of Encord platform external storage integration
+        Returns:
+            List[str].
```

Comment:
Also, from the implementation you seem to be returning `DicomDeidentifyTask`. Something is fishy here

Reply:
`DicomDeidentifyTask` is input model